server/reload.go

```diff
@@ -30,6 +30,17 @@ func (t *traceOption) Apply(server *Server) {
 	server.Noticef("Reloaded: trace = %v", t.newValue)
 }
 
+// debugOption implements the option interface for the `debug` setting.
+type debugOption struct {
+	newValue bool
+}
+
+// Apply the debug change by reconfiguring the server's logger.
+func (d *debugOption) Apply(server *Server) {
+	server.ConfigureLogger()
```

Comment:
Shouldn't be too hard to fix this, but I don't think it's worth it til we support a few options that reconfigure the logger.